### ISSUE TITLE
Actionsheet: Keyboard focus is not moving during navigation when JAWS is on and role of actionsheet is set to menu

### ISSUE THREAD

**Issue by pankajpandey01** [SAP/ui5-webcomponents#6741]
**Describe the bug**
Please see details here: https://github.com/SAP/ui5-webcomponents-react/issues/3887. 
This is reopening of above defect.

**Additional context**
Our a11y team expect ActionSheet should behave like 
https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/examples/menu-button-actions-active-descendant/

Stakeholder
Organization: {SuccessFactors}

**Comment by nnaydenow** [comment]
Hi @SAP/ui5-webcomponents-topic-rd 

Could you please check please check why the focus is not moving inside popover when the JAWS is used?

Regards,
Nayden

**Comment by LidiyaGeorgieva** [comment]
Hello @pankajpandey01 ,

The difference between the two samples  (first [codeSandbox](https://codesandbox.io/s/great-dijkstra-ekxefn?file=/src/App.js) and second [codeSandbox](https://codesandbox.io/p/sandbox/cool-sun-mi35by) ) is the position of role="application".
This difference makes Jaws to behave differently, which is not under our control.
You can see more information about this role here: https://www.a11yproject.com/posts/how-to-use-application-role/

Regards,
Lidiya

**Comment by Lukas742** [comment]
Hi @LidiyaGeorgieva 

please read what I wrote about the behavior of `role="application"`. I already outlined that screen readers behave differently when using this role, but it's not the issue here. The issue is that JAWS is ignoring the role in this case and __doesn't__ behave differently.

**Comment by Lukas742** [comment]
Hi @LidiyaGeorgieva 

applying the `role` inside the slot is working, but it's not working when applied directly on the web-component (`ui5-responsive-popover`) or somewhere else in the DOM tree (e.g. `body`) which is pretty common when using components that are fully accessible out of the box, because this way developers don't have to set the role on each component, but only in one place. 
Also, setting it inside the content of the popover (default slot), is currently not possible with our implementation as we're already applying a role there. We could add another element wrapping the content, but I would like to know why that is necessary, to prevent future problems.

**Comment by TeodorTaushanov** [comment]
Hi @Lukas742,

Unfortunately we cannot control how the screen readers virtual cursor will affect the keyboard navigation. 

I would suggest you to avoid using role="application", as It is used to denote a region of a web application that is to be treated like a desktop application.

This looks like some JAWS specific behavior. I'm not sure if this is a bug in JAWS. But you shouldn't put role="application" to avoid some screen reader issues. 

Best,
Teo

**Comment by Lukas742** [comment]
Hi @TeodorTaushanov 

I understand, thanks for the clarification.

I'll discuss with the team on how we should handle such issues in the future.
